order_publisher_node:
  ros__parameters:
    orders_path: "/home/arvuser/arv-ws/src/nav-onboarding-2024/pizza_bot_infrastructure/config/orders.json"
    repeat_orders: false